packages/outputarea/src/widget.ts

```diff
@@ -853,8 +874,11 @@ export class Stdin extends Widget implements IStdin {
       node: Private.createInputWidgetNode(options.prompt, options.password)
     });
     this.addClass(STDIN_CLASS);
+    this._history_ix = 0;
     this._input = this.node.getElementsByTagName('input')[0];
     this._input.focus();
+    // make users aware of the line history feature
+    this._input.placeholder = '↑↓ for history';
```

Comment:
This will need to be translated. But it lets do that outside of this PR because I just realized the translator object is not provided to the `Notebook.ContentFactory` nor the `Cell.ContentFactory`.